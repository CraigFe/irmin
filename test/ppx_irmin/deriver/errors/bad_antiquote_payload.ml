let _ = [%irmin: unit * [%e: val x : unit]]
