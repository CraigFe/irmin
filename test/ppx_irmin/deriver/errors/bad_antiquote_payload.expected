File "bad_antiquote_payload.ml", line 1, characters 8-43:
Error: ppx_irmin: bad payload for [%e ... ] anti-quotation. Must be a single expression.
